feature-libs/product-configurator/rulebased/core/state/effects/configurator-basic-effect.service.spec.ts

```diff
@@ -68,6 +119,28 @@ describe('ConfiguratorBasicEffectService', () => {
         classUnderTest.getFirstGroupWithAttributes(productConfiguration)
       ).toBe(GROUP_ID_8);
     });
+
+    it('should find attribute group as first group in multi level config although conflicts exist (using default value for includeConflicts)', () => {
+      productConfiguration.groups = groupListWithConflicts;
+      expect(
+        classUnderTest.getFirstGroupWithAttributes(productConfiguration)
+      ).toBe(GROUP_ID_8);
+    });
+
+    it('should find attribute group as first group in multi level config although conflicts exist if includeConflicts is set to false', () => {
+      productConfiguration.groups = groupListWithConflicts;
+      expect(
+        classUnderTest.getFirstGroupWithAttributes(productConfiguration, false)
+      ).toBe(GROUP_ID_8);
+    });
+
+    it('should find attribute group as first group in multi level config although conflicts exist if includeConflicts is set to true', () => {
```

Comment:
Please fix description of test (you expect conflict group to be found)

Reply:
Done.